### ISSUE TITLE
RK61(wired only) bricked after sometime of using rangoli

### ISSUE THREAD

**Issue by anubha-v-ardhan** [rnayabed/rangoli#30]
Hi, my RK61 keyboard is showing no sign of life (no rgb, no input, no detection in `lsusb` or even on Windows) after using Rangoli for some time. The app was working fine when suddenly the rgb lights went off and the keyboard stopped working.

Here are rangoli logs of the time it happened.
```
[25/05/2023] [18:31:10] [DEBUG] : HID Device found:  /dev/hidraw0
[25/05/2023] [18:31:10] [DEBUG] : HID Device found:  /dev/hidraw0
[25/05/2023] [18:31:10] [DEBUG] : HID Device found:  /dev/hidraw0
[25/05/2023] [18:31:10] [DEBUG] : HID Device found:  /dev/hidraw0
[25/05/2023] [18:31:10] [DEBUG] : HID Device found:  /dev/hidraw0
[25/05/2023] [18:31:10] [DEBUG] : HID Device found:  /dev/hidraw0
[25/05/2023] [18:38:53] [WARNING] : The X11 connection broke (error 1). Did the X11 server die?
[25/05/2023] [18:38:53] [FATAL] : QThread: Destroyed while thread is still running
```

I'm not sure if this issue is related to this app or it happened due to completely different reason but it would be great if I can get some assistance to fix it or some directions on why it could happen so I can research myself on fixing my keyboard.

I understand the README states no responsibility if keyboard is bricked but I'd be glad if someone can help me. Feel free to close the issue if this doesn't belong here.

Thank you.

**Comment by rnayabed** [comment]
Yes it looks like Rangoli did break your keyboard :(. The protocol is undocumented and mostly based off RK71RGB one and my assumptions. Can you provide the VID and PID of your device? or the exact model number? I will blacklist it from supported devices for future users. 

It will be great if you can try getting in touch with RK support and try rescuing your keyboard, after that you can use their official software, sniff the usb packets and send them here so that I can make changes accordingly.
